admin-tools/lib/src/main/scala/com/gu/mediaservice/BatchIndexHandler.scala

```diff
@@ -115,14 +115,6 @@ class BatchIndexHandler(cfg: BatchIndexHandlerConfig) extends LoggingWithMarkers
   }
 
   def processImages(): Unit = {
-
-    if (AwsHelpers.checkKinesisIsNiceAndFast(stage, threshold))
```

Comment:
We no longer care whether Kinesis is nice and fast – we have a low-priority stream! 🎉 

... should we keep this and extend the threshold to e.g. hours to avoid pointlessly enqueuing images when the queue is saturated? @aug24

Reply:
Discussed and dealt with.